vscode-trace-extension/src/test/extension-test.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('http://localhost:3000');
+    await page.getByRole('tab', { name: 'Welcome, preview' }).getByRole('button', { name: /Close/ }).click();
+    await page.getByRole('button', { name: 'Never' }).click();
+});
+
+test('Open Trace from Explorer', async ({ page }) => {
+    await page.getByRole('treeitem', { name: '202-bug-hunt' }).locator('a').click();
+    await page.getByRole('treeitem', { name: 'cat-kernel' }).locator('a').click({ button: 'right' });
+    await page.getByRole('menuitem', { name: 'Open with Trace Viewer' }).hover();
```

Comment:
This line fixes the intermittent failures when running it on my computer. Let's leep it in mind when implementing more tests in later commits.